client/my-sites/exporter/guided-transfer-details.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Gridicon from 'components/gridicon';
+
+const GuidedTransferDetails = ( { translate } ) => {
```

Comment:
This is not a big issue but I feel like this `GuidedTransferDetails` stuff could be in client/my-sites/exporter/guided-transfer-options.jsx instead of its own file. That was my original intention anyway.